source/dem/dem.cc

```diff
@@ -1556,7 +1604,7 @@ DEMSolver<dim>::solve()
                            triangulation,
                            particle_handler,
                            insertion_object,
-                           solids,
+                           solid_surfaces,
```

Comment:
don't you wanna add the solid volume now?

Reply:
Yes, I think it's  a good idea. This way, only the contact detection and force calculation will be missing.

Reply:
This will be done in the following PR. It is not critical right now since solid volume are not functionnal and I would like to close this PR asap, otherwise it is going to stay open for a while.